<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMTDConnectionManagerForm</class>
 <widget class="QWidget" name="VMTDConnectionManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="hl_1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbLoad">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="gbDevice">
       <property name="title">
        <string>Device</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="cbDevice"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="_6">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pbShow">
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbConnection">
       <property name="title">
        <string>Connection</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="4" rowspan="2">
         <spacer name="_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbDevice_2"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="_5">
          <property name="text">
           <string>Interface:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="_4">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="_2">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="cbInterface_2"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbDevice_1"/>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="cbInterface_1"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="_3">
          <property name="text">
           <string>Interface:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" rowspan="2">
         <widget class="QPushButton" name="pbCreate">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbConnections">
     <property name="title">
      <string>Connections</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="wConnections">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>580</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
